fabric-biomes-v1/src/testmod/java/net/fabricmc/fabric/test/biome/biome/TestCrimsonForestBiome.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.test.biome.biome;
+
+import net.minecraft.world.biome.CrimsonForestBiome;
+
+public class TestCrimsonForestBiome extends CrimsonForestBiome {
```

Comment:
This could almost jsut be an inner class in FabricBiomeTest

Reply:
Alright, done